<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>table</class>
 <widget class="QDialog" name="table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>791</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="tableBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>TableFirst</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TableTwo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TableTransactions</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="localSave">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Локальное сохранение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reportTG">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправить отчет TG</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>531</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="upperLimitRBtn">
          <property name="text">
           <string>Верхний лимит</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="lowerLimitRBtn">
          <property name="text">
           <string>Нижний лимит</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSpinBox" name="limitSpin"/>
      </item>
      <item>
       <widget class="QComboBox" name="nameLimitBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="limitBtn">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="newPosRBtn">
          <property name="text">
           <string>Новая позиция</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="upperRBtn">
          <property name="text">
           <string>Увеличение</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="lowerRBtn">
          <property name="text">
           <string>Уменьшение</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="id">
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QComboBox" name="nameBox"/>
          </item>
          <item>
           <widget class="QLineEdit" name="newName">
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSpinBox" name="cntSpin"/>
        </item>
        <item>
         <widget class="QComboBox" name="categoryBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="posBtn">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
